hack/setenv.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+
+function workflowNamespace {
+  default="sonataflow-infra"
+  read -p "Enter workflow namespace (default: $default): " value
+
+  if [ -z "$value" ]; then
+      workflow_namespace="$default"
+  else
+      workflow_namespace="$value"
+  fi
+  echo "export WORKFLOW_NAMESPACE=$workflow_namespace" > .env
+}
+
+function k8sUrl {
+  url="$(oc whoami --show-server)"
+  echo "export K8S_CLUSTER_URL=$url" >> .env
+}
+
+function k8sToken {
+  sa_namespace="orchestrator"
+  sa_name="orchestrator"
+  read -p "In which namespace we have or have to create the SA holding the persistent token? (default: $sa_namespace): " selected_ns
+  if [ -n "$sa_namespace" ]; then
+    sa_namespace="$sa_namespace"
+  fi
+
+  read -p "What is the name of the SA? (default: $sa_name): " selected_name
+  if [ -n "$selected_name" ]; then
+    sa_name="$selected_name"
+  fi
+
+  if oc get namespace "$sa_namespace" &> /dev/null; then
+    echo "Namespace '$sa_namespace' already exists."
```

Comment:
needed? I mean, the user selected the namespace earlier

Reply:
It's an extra info to better understand the script flow. I don't see a big issue in keeping it, but I can delete it if you want

Reply:
let's keep it for debug purposes and we can always remove later if needed or if we have feedback about that :)